operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -96,4 +135,4 @@ include::modules/disable-language-specific-vulnerability-scanning.adoc[leveloffs
 [id="additional-resources_examine-images-for-vulnerabilities"]
 [role="_additional-resources"]
 == Additional resources
-* For more information about Common Vulnerabilities and Exposures (CVEs), see the link:https://access.redhat.com/security/cve[Red Hat CVE Database].
+* For more information about Common Vulnerabilities and Exposures (CVEs), see the link:https://access.redhat.com/security/cve[Red Hat CVE Database].
```

Comment:
Per OpenShift documentation guidelines for [Additional resources sections](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).

> You must not include paragraphs in the section. Use an unordered list.

```suggestion
* link:https://access.redhat.com/security/cve[Red Hat CVE Database].
```

Reply:
Thanks for your feedback, but I am afraid I have not worked on this piece of content. The reason you see it under changed files is perhaps because of the removal of white spaces.

Reply:
Thanks, Agil! 
Maybe a consideration for another time if you update these files :)